src/components/ResetPasswordForm/ResetPasswordForm.tsx

```
@@ -0,0 +1,142 @@
+                    />
+                  </div>
+                );
+              }}
+            />
+          );
+        })}
+
+        <form.Subscribe
+          selector={(state) => [state.canSubmit, state.isSubmitting]}
+          children={([canSubmit, isSubmitting]) => (
+            <div className={s.buttons}>
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)